### PR TITLE
When changing a term to discrete from the scatter summary should change to violin

### PR DESCRIPTION
**Pull request by airenzp**
Fixed now

### CODE REVIEW

File: client/plots/summary.js [hunk 1]
```diff
@@ -356,20 +356,17 @@ export async function getPlotConfig(opts, app) {
 			sampleScatter: getDefaultScatterSettings(app)
 		},
 		mayAdjustConfig(config, edits = {}) {
+			console.log(config)
```
Comment (siosonel) [inline client/plots/summary.js:359]: @karishma-gangwani please remove this console.log

Comment (karishma-gangwani) [reply client/plots/summary.js:359]: @siosonel you mean Airen?

**Review by karishma-gangwani — DISMISSED** [review]

**Comment by karishma-gangwani** [comment]
thanks @airenzp

File: client/plots/summary.js (hunk 1 above)
Comment (siosonel) [reply client/plots/summary.js:359]: @karishma-gangwani yes, sorry, I meant @airenzp

Comment (airenzp) [reply client/plots/summary.js]: Done now

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**